src/NumberGuessingGame.js

```diff
@@ -13,65 +13,43 @@ function getRandomNumber() {
 
 const MAX_ATTEMPTS = 5;
 
-class NumberGuessingGame extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    };
-
-    /**
-     * These lines are required to make the methods/functions declared on this
-     *  class have the correct `this` object when they run.
-     */
-    this.handleGuess = this.handleGuess.bind(this);
-    this.handleReset = this.handleReset.bind(this);
-  }
-
-  handleGuess(guess) {
-    this.setState({
-      latestGuess: guess,
-      numberOfGuesses: this.state.numberOfGuesses + 1,
-    });
-  }
-
-  handleReset() {
-    this.setState({
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    });
+const NumberGuessingGame=()=>{ 
+  const [numberToGuess , setNumberToGuess] = useState(getRandomNumber());
```

Comment:
Nice job here 💯 
But you have a bug here -- `getRandomNumber()` gets called immediately. It should be `useState(getRandomNumber)` instead. Then only one number will be assigned to `numberToGuess`